cmake/CMakeLists.txt

```diff
@@ -1700,6 +1702,10 @@ if (onnxruntime_ENABLE_TRAINING_OPS)
   add_compile_definitions(ENABLE_TRAINING_OPS)
 endif()
 
+if (onnxruntime_ENABLE_CUDA_PROFILING)
```

Comment:
shouldn't you also check onnxruntime_ENABLE_CUDA as well?
or stick this check underneath
https://github.com/microsoft/onnxruntime/blob/ce84554ed0084150221499a7d6b5b439ce7cffba/cmake/CMakeLists.txt#L1571